apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -63,6 +67,12 @@ export const styles = {
     lineHeight: '52px',
     width: 210
   },
+  col5: {
```

Comment:
I thought about that too,  but since Brent re-uses the styles in another place I wonder if the column names should be content agnostic?